frontend/src/queries/nodes/DataVisualization/Components/Variables/NewVariableModal.tsx

```diff
@@ -175,6 +175,9 @@ export const NewVariableModal = (): JSX.Element => {
                         value={variable.name}
                         onChange={(value) => updateVariable({ ...variable, name: value })}
                     />
+                    {modalType === 'new' && variable.name.length > 0 && (
+                        <span className="text-xs">{`Use this variable by referencing {variables.${variable.name.replaceAll(' ', '_')}}.`}</span>
```

Comment:
Thanks for this - one suggestion, it's not just a simple `' '` replace to `'_'` - this is what we do on the backend in python:

```python
"".join(n for n in validated_data["name"] if n.isalnum() or n == " ").replace(" ", "_").lower()
```

Reply:
Hi @Gilbert09, thanks for the review + suggestion!